demos/iCubTest/composeGui.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.7"
+
+x-icub-test: &icub-test
+  image: icubteamcode/superbuild-icubtest:master-unstable_sources
+  environment:
+    - DISPLAY=${DISPLAY}
+    - QT_X11_NO_MITSHM=1
+    - XAUTHORITY=/root/.Xauthority
+    - YARP_FORWARD_LOG_ENABLE=1
+    - YARP_ROBOT_NAME
+    - SUITE_TYPE
+    - CUSTOM_SUITE
+    - CUSTOM_FILE_PATH
+    - RESULTS_PATH
+  volumes:
+    - "/tmp/.X11-unix:/tmp/.X11-unix"
+    - "${XAUTHORITY}:/root/.Xauthority"
+    - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
+    - "${RESULTS_PATH:-/tmp/icub-test_results}:/projects/robotology-superbuild/src/icub-tests/suites/results:rw"
```

Comment:
This folder should be `/tmp/empty-dir` as in [here](https://github.com/icub-tech-iit/appsAway/pull/322/files#diff-7ca6eddeb050d8c18039d836de573c806b1f000be1be3dff53b9693fd6a9b5e4R15-R16), I don't see it changed in the deployment correct?

Reply:
We assigned this default value for the environment variable along with @valegagge. The `empty-dir` folder has problems with managing permissions and deleting files after the first RUN of the application, whereas with that folder we are able to better manage data storage.

In any case, these volumes should only be present in the `composeGui.yml` and I did a commit to remove these two lines.